src/components/PageHeader.tsx

```diff
@@ -62,7 +62,7 @@ export const PageHeader = ({ title }: Props) => {
         </div>
         <a
           className={classes.contact}
-          href={FIDER_LINK}
+          href={process.env.REACT_APP_FIDER_LINK}
```

Comment:
You also need to remove the import at the top:  { FIDER_LINK } from 'utils/constants', and remove FIDER_LINK from constants.

Reply:
Removed the import.